documentation/getstarted/firststeps/commandline.md

```diff
@@ -7,26 +7,41 @@ redirect_from: /getStarted/firststeps/commandline/
 # First Steps: Command-line
 
 This brief tutorial will teach **how to get up and running with the Flyway Command-line tool**. It will take you through the
-steps on how to configure it and how to write and execute your first few database migrations.
+steps on how to configure it and how to write and execute your first few database migrations, using <a href="spawn.cc">Spawn</a> to provision a database instance without the need for installing any database engines onto your own machine. You can create MySQL, MSSQL and PostgreSQL databases with Spawn that will work with Flyway.
 
-This tutorial should take you about **5 minutes** to complete.
+This tutorial should take you about **10 minutes** to complete.
 
-## Downloading and extracting Flyway
+## Prerequisites
 
 Start by <a href="/download">downloading the Flyway Command-line Tool</a> for your platform and extract it.
 
-Let's jump into our new directory:
-
-<pre class="console"><span>&gt;</span> cd flyway-{{ site.flywayVersion }}</pre>
+Install Spawn by visiting the <a href="https://www.spawn.cc/docs/getting-started.html">getting started documentation</a> and following the installation steps.
 
 ## Configuring Flyway
 
-Once that is done, configure Flyway by editing `/conf/flyway.conf` like this:
+To configure Flyway, we first need a database we can connect to. We’ll use Spawn to create your own, isolated database environment from which you can run migrations against. This will create you your first data container, which is your database instance. Here we are specifying a PostgreSQL database but you can also specify `mysql:empty` or `mssql:empty`:
+
+<pre class="console"><span>&gt;</span> spawnctl create data-container --image postgres:empty --name flyway-container</pre>
+
+This will return connection string details which is used to connect and query using your normal tools:
+
+<pre class="console">Data container 'flyway-container' created!
+-> Host=instances.spawn.cc;Port=xxxxx;Username=xxxx;Database=foobardb;Password=xxxxxxxxx</pre>
+
+You can retrieve these details at any time by running:
+
+<pre class="console"><span>&gt;</span> spawnctl get data-container flyway-container -o yaml</pre>
+
+Let's now jump into our new directory created from downloading Flyway:
+
+<pre class="console"><span>&gt;</span> cd flyway-{{ site.flywayVersion }}</pre>
+
+Configure Flyway by editing `/conf/flyway.conf` with your Spawn data container connection details, like this:
 
 ```properties
-flyway.url=jdbc:h2:file:./foobardb
-flyway.user=SA
-flyway.password=
+flyway.url=jdbc:postgresql://instances.spawn.cc:port/foobardb
+flyway.user=user
+flyway.password=password
```

Comment:
Do `port`, `user` and `password` here correspond to `Port`, `Username` and `Password` from line 29?

If so, I think it would be better to write them as `<Port>`, `<Username>` and `<Password>` to make it more obvious that they are to be substituted in and not actual values